scripts/parallel.py

```diff
@@ -139,30 +139,33 @@ def main(cfg: DictConfig):
     # For now, all configs are the same
     envs.call("update_config", env_cfg, reset=False)
     # Need separate seeds for each environment
-    rkey = utils.JaxRKey(seed)
-    seeds = [rkey.next_seed() for _ in range(world_draws)]
-
+    seeds = [seed] * world_draws
+    observations, infos = envs_wrapper.reset(seed=seeds)
+    
+    mc_seeds = [random.randint(0, 2**32 - 1) for _ in range(world_draws)]
```

Comment:
We should instead keep the `rkey` and use it as
```
mc_seeds = [rkey.next_seed() for _ in range(world_draws)]
```
You can think of the `rkey` as a RNG int generator.